source/templates/conditionals.md

```diff
@@ -1,7 +1,49 @@
-Sometimes you may only want to display part of your template if a property
-exists.
+Conditional statements bring a minimum of logic into Ember templating.
+Statements like `if` and `unless` are implemented as built-in helpers. Helpers
```

Comment:
At this point in the Templates section, the term "helpers" hasn't been defined yet.